AsyncPoco.Tests/App.config

```diff
@@ -3,7 +3,7 @@
 	<connectionStrings>
 		<add name="mysql" connectionString="server=localhost;database=asyncpoco;user id=asyncpoco;password=asyncpoco;Allow User Variables=true" providerName="MySql.Data.MySqlClient" />
 		<add name="sqlserver" connectionString="Data Source=localhost;Initial Catalog=AsyncPocoTest;Integrated Security=True" providerName="System.Data.SqlClient" />
-		<add name="sqlserverce" connectionString="Data Source=petapoco.sdf" providerName="System.Data.SqlServerCe" />
+		<add name="sqlserverce" connectionString="Data Source=|DataDirectory|\petapoco.sdf" providerName="System.Data.SqlServerCe" />
```

Comment:
After converting to use NUnit, I found that I needed to fully qualify the Data Source for the sqlserverce connection. The use of |DataDirectory| is a standard practice when doing this.